webapp/src/App.js

```diff
@@ -62,14 +74,8 @@ class DecksList extends React.Component {
       return <div>Loading...</div>;
     } else {
       return (
-        <List component="ul">
-          {
-            decks.map(
-              (deck) => {
-                return (<DecksListDeck deck={deck} key={deck.id} api={this.props.api} />);
-              }
-            )
-          }
+        <List dense={false}>
+          {decks.map((deck) => { return <DecksListDeck key={deck.id} deck={deck} deleteDeck={this.deleteDeck.bind(this)} /> })}
```

Comment:
It turns out we can just bind `this` in the event handler function this way.

Reply:
This page is a good short read on handling events in Reactjs and binding in Javascript: https://reactjs.org/docs/handling-events.html